cpp/velox/shuffle/VeloxSplitter.h

```diff
@@ -385,6 +390,9 @@ class VeloxSplitter {
 
   // shared by all partition writers
   std::shared_ptr<arrow::ipc::IpcPayload> schema_payload_;
+
+  facebook::velox::DecodedVector decoded_vector_;
```

Comment:
Each column should keep one `DecodedVector` as here do https://github.com/facebookincubator/velox/blob/main/velox/functions/prestosql/aggregates/ApproxDistinctAggregate.cpp#L377

Reply:
the DecodedVector is used to access  the wrapped Vector's data for the purpose of performance, and the `decode()` will call `reset()`, it's just a shell and `decode()` will reuse it.

Reply:
you can check its help document [DecodeVector](https://facebookincubator.github.io/velox/develop/dictionary-encoding.html#decodedvector)

Reply:
https://facebookincubator.github.io/velox/develop/aggregate-functions.html